File: src/main/java/cairo/psi/CairoTokenType.java
package cairo.psi;

import com.intellij.psi.tree.IElementType;
import org.jetbrains.annotations.NonNls;
import org.jetbrains.annotations.NotNull;
import cairo.CairoLanguage;

public class CairoTokenType extends IElementType {
	public CairoTokenType(@NotNull @NonNls String debugName) {
		super(debugName, CairoLanguage.INSTANCE);
	}
}

File: src/main/kotlin/cairoplugin/language/psi/CairoTokenType.kt
package cairoplugin.language.psi

import cairoplugin.language.CairoLanguage
import com.intellij.psi.tree.IElementType
import org.jetbrains.annotations.NonNls
import org.jetbrains.annotations.NotNull

class CairoTokenType(@NotNull @NonNls debugName: String) : IElementType(debugName, CairoLanguage.INSTANCE) {

    override fun toString(): String {
        return "CairoTokenType." + super.toString()
    }
}